### PR TITLE
tutorial-bug: Clarify that suite won't run _yet_

### PR DESCRIPTION
**Pull request by wxtim**
I don't think it's clear that this suite won't run yet, and that we are only testing the installation and validation at this point.

### CODE REVIEW

File: sphinx/tutorial/rose/suites.rst [hunk 1]
```diff
@@ -416,8 +416,9 @@ See the :ref:`Cheat Sheet` for more information.
       Running :ref:`command-rose-suite-run` will cause the suite to be
       installed, validated and run.
```
Comment (oliver-sanders) [inline sphinx/tutorial/rose/suites.rst:417]:
```suggestion
```

Comment (wxtim) [reply sphinx/tutorial/rose/suites.rst]: I hadn't been going to be that brutal, but yes, that works!

**Review by oliver-sanders — APPROVED** [review]

**Comment by oliver-sanders** [comment]
No need to copy these changes to cylc-doc as this section will be re-written.

**Comment by oliver-sanders** [comment]
Going to ignore test failures as Rose2 needs a bit of tlc.

**Pull request merged**